official/resnet/resnet_run_loop.py

```diff
@@ -408,6 +408,9 @@ def input_fn_eval():
     if benchmark_logger:
       benchmark_logger.log_estimator_evaluation_result(eval_results)
 
+  if benchmark_logger:
```

Comment:
This feels very inelegant to me, but I haven't come up with a better solution, at least assuming that explicit >> implicit.